XamlAnimatedGif/Animator.cs

```diff
@@ -519,17 +516,15 @@ private async Task<Stream> GetIndexStreamAsync(GifFrame frame, CancellationToken
             return lzwStream;
         }
 
-        private async Task<byte[]> GetIndexBytesAsync(int frameIndex, CancellationToken cancellationToken)
+        private async Task GetIndexBytesAsync(int frameIndex, byte[] buffer,  CancellationToken cancellationToken)
```

Comment:
Why the cancellation token? This is only called in one place, and you pass `CancellationToken.None`.

Reply:
I removed the extra parameter